### ISSUE TITLE
Could not import extension rinoh.frontend.sphinx

### ISSUE THREAD

**Issue by dcalvin** [brechtm/rinohtype#122]
Subject: <No module named rinoh.frontend.sphinx>

### Problem
failed to build PDF after I firstly installed rinohtype and used it with Sphinx. Begging for help. Thanks in advance.

#### Procedure to reproduce the problem
# -- Command Prompt-----

D:\Sphinx for HLTAPI_py3\source>sphinx-build -b rinoh . _build/rinoh
Running Sphinx v1.6.3
loading translations [en]... done

Extension error:
Could not import extension rinoh.frontend.sphinx (exception: No module named rinoh.frontend.sphinx)

# -- Options  in conf.py---------------------------------------------

extensions = ['sphinx.ext.autodoc','rst2pdf.pdfbuilder', 'rinoh.frontend.sphinx']

rinoh_documents = [('index',            # top-level file (index.rst)
                    'HLTAPI',           # output (target.pdf)
                    'Spirent TestCenter Automation HLTAPI Command Reference',   # document title
                    'XTAPI')]   # document author 

 
### Environment info
- OS:  Windows 10
- Python version: 3.6.4
- Sphinx version: 1.6.3
- Docutils version: 0.14

**Comment by brechtm** [comment]
Were you able to solve the issue?

**Comment by dcalvin** [comment]
Thanks for confirming. I solved this issue, which was caused by the calling the Sphinx module (build 1.6.3) from Python 2, while rinoh is deployed with Python 3 (together with the latest Sphinx build 1.7.1).. While I ran the rinoh  commands in the environment configured by the latter, it still called Sphinx 1.6.3, which would not be able to identify the rinoh  extension.  After I uninstalled  Sphinx in Python 2, it worked.
